src/run_all.py

```diff
@@ -97,7 +97,7 @@ def main(
         "antecedent_dry_days": 1,
         "average_rain_intensity": 1,
         "event_duration": 1,
-        "rainfall_ph": 7
+        "rainfall_ph": 8
```

Comment:
```suggestion
        "rainfall_ph": 7
```
Keeping the PR changes as small as possible is good, so I am suggesting changing this back to 7